src/utils/bindtwostepcarettoattribute.js

```diff
@@ -122,13 +127,21 @@ export default function bindTwoStepCaretToAttribute( view, model, emitter, attri
 		const position = modelSelection.getFirstPosition();
 		let isMovementHandled;
 
-		if ( arrowRightPressed ) {
-			isMovementHandled = twoStepCaretHandler.handleForwardMovement( position, data );
+		if ( contentDirection === 'rtl' ) {
```

Comment:
I feel this will be more readable:

```js
if ( ( contentDir == ltr && arrowRight ) || ( contentDir == rtl && arrowLeft ) ) { 
 handleForward();
} else {
  handleBackward();
}
```

It's more straighforward that there are only two logical cases we're handling here – forward and backward moves. There are no other options. And in the current implementation that's actually not that obvious.